packages/sources/kaiko/README.md

```diff
@@ -47,4 +48,33 @@ There are no examples for this endpoint.
 
 ---
 
+## Realized-vol Endpoint
+
+Supported names for this endpoint are: `realized-vol`, `realized-volatility`.
+
+### Input Params
+
+| Required? |    Name    |          Aliases          |                       Description                        |  Type  | Options | Default  | Depends On | Not Valid With |
+| :-------: | :--------: | :-----------------------: | :------------------------------------------------------: | :----: | :-----: | :------: | :--------: | :------------: |
+|    ✅     |    base    |      `coin`, `from`       |  The base currency to query the realized volatility for  | string |         |          |            |                |
+|           |   quote    | `convert`, `market`, `to` | The quote currency to convert the realized volatility to | string |         |  `USD`   |            |                |
+|           | resultPath |                           |        The field to return within the result path        | string |         | `result` |            |                |
+
+### Example
+
+Request:
+
+```json
+{
+  "data": {
+    "endpoint": "realized-vol",
+    "base": "ETH",
+    "quote": "USD",
+    "resultPath": "result"
```

Comment:
I think `resultPath` needs to be one of `['realVol1Day', 'realVol7Day', 'realVol30Day']`

Reply:
yeah, I think i need to regenerate these. 🙏